### PR TITLE
fix: log unknown Broadcast errors

### PR DESCRIPTION
**Pull request by chasers**
- adds a catchall in the FallbackController and logs unknown errors

### CODE REVIEW

File: lib/realtime_web/controllers/fallback_controller.ex [hunk 1]
```diff
@@ -54,4 +54,13 @@ defmodule RealtimeWeb.FallbackController do
     |> put_view(RealtimeWeb.ChangesetView)
     |> render("error.json", changeset: changeset)
   end
+
+  def call(conn, response) do
+    Helpers.log_error("UnknownError",response)
+
+    conn
+    |> put_status(:not_found)
```
Comment (filipecabaco) [inline lib/realtime_web/controllers/fallback_controller.ex:62]: this might be better as a 422 and not a 404

**Review by filipecabaco — APPROVED** [review]

File: lib/realtime_web/controllers/fallback_controller.ex (hunk 1 above)
Comment (chasers) [reply lib/realtime_web/controllers/fallback_controller.ex:62]: That is for the TenantController for when tenants are not found though

Comment (chasers) [reply lib/realtime_web/controllers/fallback_controller.ex:62]: Oh...

**Pull request merged**